<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWindow</class>
 <widget class="QMainWindow" name="PlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SuperSonic Audiobook Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>563</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="cover">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>350</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../assets/defaultBookCover.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>Title</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="authorLabel">
         <property name="text">
          <string>Author</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="narratorLabel">
         <property name="text">
          <string>Narrator</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="jumpBackButton">
           <property name="text">
            <string>|&lt;</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="skipBackButton">
           <property name="text">
            <string>&lt;&lt;</string>
           </property>
           <property name="shortcut">
            <string>Left</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pauseButton">
           <property name="text">
            <string>||</string>
           </property>
           <property name="shortcut">
            <string>Space</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="skipForwardButton">
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
           <property name="shortcut">
            <string>Right</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="jumpForwardButton">
           <property name="text">
            <string>&gt;|</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+Right</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QDoubleSpinBox" name="speedControl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="suffix">
          <string>x</string>
         </property>
         <property name="minimum">
          <double>0.050000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="textDirection">
          <enum>QProgressBar::TopToBottom</enum>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="volumeControl">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControls">
    <property name="title">
     <string>Controls</string>
    </property>
    <addaction name="actionPause"/>
    <addaction name="actionSkipForward"/>
    <addaction name="actionSkipBack"/>
    <addaction name="actionJumpForward"/>
    <addaction name="actionJumpBack"/>
    <addaction name="separator"/>
    <addaction name="actionVolumeUp"/>
    <addaction name="actionVolumeDown"/>
    <addaction name="actionFaster"/>
    <addaction name="actionSlower"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuControls"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pause/Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionSkipForward">
   <property name="text">
    <string>Skip Ahead</string>
   </property>
   <property name="toolTip">
    <string>Skip forward 15 seconds</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionSkipBack">
   <property name="text">
    <string>Skip Back</string>
   </property>
   <property name="toolTip">
    <string>Skip Backward </string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionJumpForward">
   <property name="text">
    <string>Jump Ahead</string>
   </property>
   <property name="toolTip">
    <string>Jump forward a chapter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionJumpBack">
   <property name="text">
    <string>Jump Back</string>
   </property>
   <property name="toolTip">
    <string>Jump back a chapter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open a new audio file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionVolumeUp">
   <property name="text">
    <string>Volume Up</string>
   </property>
   <property name="toolTip">
    <string>Raise the volume</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionVolumeDown">
   <property name="text">
    <string>Volume Down</string>
   </property>
   <property name="toolTip">
    <string>Lower the volume</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionFaster">
   <property name="text">
    <string>Faster</string>
   </property>
   <property name="toolTip">
    <string>Increase the playback speed</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionSlower">
   <property name="text">
    <string>Slower</string>
   </property>
   <property name="toolTip">
    <string>Decrease the playback speed</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Close the program</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionPause</sender>
   <signal>toggled(bool)</signal>
   <receiver>pauseButton</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>PlayerWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionPause</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
